frontend/src/components/USAMap.js

```
@@ -1,175 +1,216 @@
-'use client';
+"use client";
 
-import { useState, useCallback } from 'react';
-import mapboxgl from 'mapbox-gl';
-import Map, { Source, Layer } from 'react-map-gl';
-import 'mapbox-gl/dist/mapbox-gl.css';
+import { useState, useEffect, useCallback } from "react";
+import mapboxgl from "mapbox-gl";
+import Map, { Source, Layer } from "react-map-gl";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)